xsd/siri_model/siri_situationActions.xsd

```diff
@@ -599,7 +600,7 @@ Rail transport, Roads and road transport
      </xsd:element>
      <xsd:element name="TextualContent" type="TextualContentStructure" maxOccurs="unbounded">
       <xsd:annotation>
-       <xsd:documentation>All texts of the passenger information.</xsd:documentation>
+       <xsd:documentation>The actual, structured passenger information.</xsd:documentation>
```

Comment:
add "in a specific language"?

Reply:
Actually, we were intending to provide several languages in a TextualContent

maybe we could add that it can be designated for a specific TextualContentSize, would this help?
![image](https://user-images.githubusercontent.com/1889203/122420379-f0114700-cf8b-11eb-8089-4a23bbe6e33d.png)

Reply:
You are correct. The reason for this 'maxOccurs=unbounded' is for varying the TextualContentSize, not the language. I should have said:  add "for a specific TextualContentSize".